src/promptflow-core/promptflow/core/_flow.py

```diff
@@ -469,6 +470,24 @@ def render(self, *args, **kwargs):
         # For chat mode, the message generated is list type. Convert to string type and return to user.
         return str(prompt)
 
+    def estimate_token_count(self, *args, **kwargs):
+        """Estimate the token count. Return the sum of prompt tokens and max tokens.
```

Comment:
add the reason of this is openai will reject request based on sum of request(prompt)+ reponse(max_token().

Reply:
update the comment